File: app/src/main/java/com/mdkashif/alarm/custom/CustomProgressDialog.java
package com.mdkashif.alarm.custom;

import android.app.Dialog;
import android.content.Context;
import android.graphics.Color;
import android.graphics.drawable.ColorDrawable;
import android.os.Bundle;
import android.view.Window;

import com.mdkashif.alarm.R;

import androidx.annotation.NonNull;

/**
 * Created by Kashif on 17-Apr-18.
 */
public class CustomProgressDialog extends Dialog {
    Context context;
    public CustomProgressDialog(@NonNull Context context) {
        super(context);
        this.context =context;
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.layout_dialog_custom_progress);
        setCanceledOnTouchOutside(false);
        setCancelable(false);
        getWindow().setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
    }
}

File: app/src/main/java/com/mdkashif/universalarm/custom/CustomProgressDialog.kt
package com.mdkashif.universalarm.custom

import android.app.Dialog
import android.content.Context
import android.graphics.Color
import android.graphics.drawable.ColorDrawable
import android.os.Bundle
import android.view.Window

import com.mdkashif.universalarm.R

/**
 * Created by Kashif on 17-Apr-18.
 */
class CustomProgressDialog(context: Context) : Dialog(context) {

    override fun onCreate(savedInstanceState: Bundle) {
        super.onCreate(savedInstanceState)
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.layout_dialog_custom_progress)
        setCanceledOnTouchOutside(false)
        setCancelable(false)
        window!!.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
    }
}
